catalog/spatial/wfs/spatial-wfs-featuretransformer/src/main/java/org/codice/ddf/spatial/ogc/wfs/featuretransformer/impl/WfsTransformerProcessor.java

```diff
@@ -55,7 +55,7 @@ public void setActiveFeatureMemberNodeName(WfsMetadata metadata, String featureM
           return metacardOptional;
         }
       } catch (IOException e) {
-        LOGGER.error(
+        LOGGER.debug(
```

Comment:
❓ Is it your intent that his exception be caught silently whenever debugging is not enabled in the logger?

Reply:
Yes, `debug` is the correct log level for exceptions like this. `error` is definitely not the right level: https://codice.atlassian.net/wiki/spaces/DDF/pages/7045131/Log+Levels